### PR TITLE
Randombeer

### PR DESCRIPTION
**Pull request by pasin7793**
RandomBeer 부분

### CODE REVIEW

File: BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+//  Created by 임준화 on 2022/06/23.
+//
+
+import Foundation
+import Alamofire
+import UIKit
+
+final class RandomBeerViewModel{
+    private let urlString = "https://api.punkapi.com/v2/beers/random"
+    var dataSource: Beer?
```
Comment (baekteun) [inline BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift:14]: 얘 뭐여

Comment (pasin7793) [reply BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift:14]: ?? 무슨의미

Comment (baekteun) [reply BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift:14]: dataSource가 왜 있어

Comment (pasin7793) [reply BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift]: https://github.com/pasin7793/BBB/pull/3/commits/449800bdc3fd41556a1322844adc9bc59d42d7bc

File: BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+final class RandomBeerViewModel{
+    private let urlString = "https://api.punkapi.com/v2/beers/random"
+    var beer: Dynamic<Beer?> = Dynamic(nil)
+    
+    func fetchData() {
+        AF.request(urlString).responseData { (response) in
+            switch response.result {
+            case .success(let res):
+                let decoder = JSONDecoder()
```
[the remark this answers is not in the document]
Comment (pasin7793) [reply BBB/Source/Item/RandomBeer/ViewModel/RandomBeerViewModel.swift:20]: https://github.com/pasin7793/BBB/pull/3/commits/84944dae00392e46fa596abde4830ba7a897b34e

File: BBB/Source/Item/BeerSearch/ViewModel/BeerSearchViewModel.swift [hunk 3]
```diff
@@ -13,13 +13,13 @@ final class BeerSearchViewModel{
     
     private let urlString = "https://api.punkapi.com/v2/beers"
     var beer: Dynamic<Beer?> = Dynamic(nil)
+    let decoder = JSONDecoder()
```
Comment (baekteun) [inline BBB/Source/Item/BeerSearch/ViewModel/BeerSearchViewModel.swift:16]: private

**Review by baekteun — COMMENTED** [review]
그리고 beerSearch부분도

File: BBB/Source/Item/BeerSearch/ViewModel/BeerSearchViewModel.swift (hunk 3 above)
Comment (pasin7793) [reply BBB/Source/Item/BeerSearch/ViewModel/BeerSearchViewModel.swift]: https://github.com/pasin7793/BBB/pull/3/commits/46dd5ad05a3cdb36b5323d7385766d006694b185

Comment (pasin7793) [reply BBB/Source/Item/BeerSearch/ViewModel/BeerSearchViewModel.swift]: https://github.com/pasin7793/BBB/pull/3/commits/6f1b048cc4c959fe3f398890f343c4c5131aa306

File: BBB/Source/Item/RandomBeer/RandomBeerVC.swift [hunk 4]
```diff
@@ -1,10 +1,76 @@
...
+    private let idLabel = UILabel().then{
+        $0.textColor = .black
+        $0.font = UIFont(name: "Helvetica", size: 12)
+    }
+    private let descriptionLabel = UILabel().then{
+        $0.font = UIFont(name: "Helvetica-bold", size: 16)
+        $0.textAlignment = .center
+    }
+    
+    private let ResetButton = UIButton().then{
```
Comment (baekteun) [inline BBB/Source/Item/RandomBeer/RandomBeerVC.swift:23]: 네이밍

Comment (pasin7793) [reply BBB/Source/Item/RandomBeer/RandomBeerVC.swift]: https://github.com/pasin7793/BBB/pull/3/commits/7402fdee0932c69656621065e9e041a9662900a6

**Review by baekteun — APPROVED** [review]

**Pull request merged**